### PR TITLE
Minor post release improvements

### PR DESCRIPTION
**Pull request by vesln**
#### Typekit on Options page

Typekit was not being loaded on the options page, therefore the font was not Proxima Nova. That's been fixed.

#### Logo as inline svg

The logo is now inlined inside the HTML to avoid loading errors when opening and going to some url quickly.

#### Post screenshot clickthrough

When a user clicks on the screenshot, we now take them directly to the product page.

#### Post card - more room

Added more height to the product "cards", so when the description is longer, it doesn't look bad.

#### Disabled Default Tab view

Instead of showing a blank page, show a disabled page. Look at the trello card for more information about that.

#### Misc

- Improve test coverage (both unit and acceptance tests)
- Use environment specific names for the extension, so it's easily distinguished which version you are using (production, staging, development)
- Enable verbose mode for `gulp * --watch`, so you get some feedback on what has been changed
- Improve debugging by introducing more `debug` calls
- (npm) Dependency cleanup
- Introduce environment dependent minification
- Introduce a gulp task for creating a package of the extension
- `jest-cli` now works just fine, useful when you want to run a single file
- Introduce `EXT_ENV` (different from `NODE_ENV`)
- Normalize file naming (to follow what's being used in the webapp)
- Normalize internal naming (post instead of product)
- Refactoring

## Notes

This could've been multiple small PRs, but combining all of them will make reviewing of the next version easier.

### CODE REVIEW

File: src/apps/options/main.js [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
 
 /**
  * Dependencies.
  */
 
-let Options = require('./components/options.react');
+let Options = require('./components/Options.react');
+
+// load Typekit
+loadTypekit();
```
Comment (andreasklinger) [inline src/apps/options/main.js]: FYI we got now an license for self hosted fonts - you can fetch it from the webapp `/app/assets/fonts`

File: src/apps/tabs/components/Post.react.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            {post.votes_count}
+          </div>
+
+          <div className="details">
+            <h3><a onClick={this._openPost}>{post.name}</a></h3>
+            <p>{post.tagline}</p>
+          </div>
+
+          <div className="comments">
+            {post.comments_count}
```
Comment (andreasklinger) [inline src/apps/tabs/components/Post.react.js]: imho everything clicks to the website apart of the comments count and it's icon (with different highlight)

File: src/common/analytics/tracker.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+   * @param {Function} callback
+   * @private
+   */
+
+  _getAnonymousId(cb) {
+    this.storage.get({ userId: null }, (items) => {
+      if (items.userId) {
+        debug('User ID found in cache');
+        return cb(items.userId);
+      } else {
```
Comment (RStankov) [inline src/common/analytics/tracker.js]: You don't need the else.

File: src/apps/tabs/components/Post.react.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  },
+
+  /**
+   * Handle post click events.
+   *
+   * @param {Object} event
+   */
+
+  _onClick(e) {
+    analytics.clickPost(this.props.post);
```
Comment (andreasklinger) [inline src/apps/tabs/components/Post.react.js]: are you sure you want the same analytics event here?

File: src/common/render/__tests__/index-test.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+jest.autoMockOff();
+
+describe('render', function() {
+  let React = require('react');
+  let render = require('../');
```
Comment (RStankov) [inline src/common/render/__tests__/index-test.js]: This is a bit confusing. Is it the same as `require('.../index.js')`?

Comment (vesln) [inline src/common/render/__tests__/index-test.js]: Right

File: src/apps/tabs/components/Post.react.js (hunk 4 above)
Comment (vesln) [inline src/apps/tabs/components/Post.react.js]: We've got two analytics events at this point - post click on the index page (default tab) and product bar click, so yeah, should be the same.

**Comment by vesln** [comment]
Feedback addressed

**Pull request merged**